tests/datastore_pyelasticsearch/tox.ini

```
@@ -0,0 +1,26 @@
+[tox]
+setupdir = {toxinidir}/../..
+envlist =
+    py27,py35,py36,py37,py38,py39,pypy,pypy3
```

```suggestion
   py27,py36,pypy
```

To match [pyelasticsearch](https://github.com/pyelasticsearch/pyelasticsearch/blob/7c2d3dd7dba80e36f061cd20194287ba92b8f81f/tox.ini#L7)